server/src/handlers/influxdb.rs

```diff
@@ -81,6 +82,135 @@ impl From<Bytes> for WriteRequest {
 
 pub type WriteResponse = ();
 
+/// Influxql query request compatible with influxdb 1.8
+///
+/// It's derived from query string parameters in influxdb 1.8:
+///     https://docs.influxdata.com/influxdb/v1.8/tools/api/#query-string-parameters-1
+///
+/// NOTE:
+///     - when query by POST method, query(q) should be placed after
+///       `--data-urlencode` like what shown in link above.
+///     - when query by GET method, query should be placed in url
+///       parameters(where `db`, `epoch`, etc are placed in).
+#[derive(Debug)]
+pub struct InfluxqlRequest {
+    pub query: String,
+    // TODO: make use of the parameters.
+    pub chunked: Option<usize>,
```

Comment:
I suggest we remove unused params.

Or you can add comments explain which params are used now.